libs/libvtrutil/src/vtr_bimap.h

```diff
@@ -92,6 +109,37 @@ class bimap {
     ///@brief Return true if the specified value exists
     bool contains(const V value) const { return find(value) != inverse_end(); }
 
+  public: //Constructors/Deconstructor
+    ///@brief default constructor required by compiler
+    bimap() = default;
+
+    ///@brief default deconstructor required by compiler
+    ~bimap() = default;
+
+    ///@brief copy constructor
+    bimap(const bimap<K, V, Map, InvMap>& other)
+        : map_(other.map_)
+        , inverse_map_(other.inverse_map_) {}
+
+    ///@brief move constructor
+    bimap(const bimap<K, V, Map, InvMap>&& other) {
+        map_ = std::move(other.map_);
+        inverse_map_ = std::move(other.inverse_map_);
+        this->clear();
```

Comment:
the `clear()` call is a bit surprising - does it mean you end up with an empty bimap after a move ? Is this intended ?

Reply:
I'd not call `other.clear()`. After the move, the `other` object is essentially in an undefined state. So just leave it as is.

Reply:
That makes sense, thanks. As requested, I have used the default copy and move constructors and removed these definitions. So there is no `clear()` call anymore in the next update.